ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Issue description
+
+
+
+
```

Comment:
This space should make it more obvious that you can fill in the actual issue here